### ISSUE TITLE
Unable to connect/deploy/debug from Visual Studio to HoloLens 2 device since Windows Holographic 24H1 Oct. 24 update

### ISSUE THREAD

**Issue by emaschino** [microsoft/OpenXR-SDK-VisualStudio#136]
Hi,

Just in case someone hit the same issue, I was suddenly unable to deploy an app to my HoloLens 2 device from Visual Studio 2017 (also tested with VS 2019). Classic error DEP6957 (BootstrapForDugging returned null instead of IPAdress & PortNumber of the remote msvsmon process). Alternate authentication method didn't work either. My development environment didn't change but I've noticed that my HL2 device was waiting for a reboot to complete update to Windows Holographic 24H1 Nov. 24 build 22621.1409. No better once rebooted.

Since my development environment didn't change, maybe the Windows Holographic update was involved. With the help of the Advanced Recovery Companion, I've thus reinstalled older Windows Holographic updates to my HL2 and was able to fix the aforementioned deployment problem.

To summarize: The last working version was Sep. 24 build 22621.1399. The regression (feature?) was introduced with Oct. 24 build 22621.1402. The latest (and final) feature update Nov. 24 build 22621.1409 didn't fix the problem.

Unfortunately, I'm unable to prevent my HL2 device to install updates, only delay their install for 35 days. With the demise of the HL2, I don't know if MS is still reading this forum. If there's a way to block any OS update, please let me know. For completeness, I'm stuck with VS 2017/2019 ATM, I cannot update to VS 2022 if this is the modern way to go.

Hope this help.

**Comment by emaschino** [comment]
I had the opportunity to test with a system with Visual Studio 2022 installed: exact same problem.

Hope a future Windows Holographic update will address this issue in a timely manner.

**Comment by emaschino** [comment]
Comments on Mixed Reality Blog [[1](https://techcommunity.microsoft.com/blog/themixedrealityblog/what%E2%80%99s-new-in-windows-holographic-version-24h1/4097407)] have been closed, so can't update there. Good news is that Windows Holographic 24H1 Jan. 25 build 22621.1418 has fixed the issue. Since I can't find any mention in the release notes, I don't know if it was resolved in this version or the previous 24H1 Dec. 24 build 22621.1413 that I didn't have time to test.
